Add 'walkthrough' to debugger settings (#8331)
microsoft/vscode-docs: docs/csharp/debugger-settings.md

You can configure the C# debugger in Visual Studio Code with a `launch.json`, `launchSettings.json`, or your user `settings.json` file.

> ## Walkthrough: setting command line arguments

Reviewer: ```suggestion
## Walkthrough: setting command-line arguments
```

> ## Walk through: setting command line arguments

Reviewer 2: Should this be `Walkthrough` (without a space)?

Author: Yes

> Before we get into the details of all the possible options, lets walkthrough a basic scenario -- setting command line arguments to your program. These steps will also work for updating other basic options like environment variables or the current working directory.

Reviewer: ```suggestion
Before we get into the details of all the possible options, let's walk through a basic scenario: setting command-line arguments to your program. These steps also work for updating other basic options like environment variables or the current working directory.
```

### Approach 1: `launchSettings.json`

> For C# Dev Kit, the recommended way to debug is to let C# Dev Kit automatically figure out how to debug from settings in the project file. This means you either don't have a `<workspace_root>/.vscode/launch.json` file, or if you have one you have `"type": "dotnet"` set for the active configuration. For command line arguments, "figure out from the project file" means to pull the value from `<Project-Directory>/Propeties/launchSettings.json`. The advantage of `launchSettings.json` is that it allows settings to be shared between Visual Studio Code, full Visual Studio, and `dotnet run`.

Reviewer: ```suggestion
For C# Dev Kit, the recommended way to debug is to let C# Dev Kit automatically figure out how to debug from settings in the project file. This means that you either don't have a `<workspace_root>/.vscode/launch.json` file, or if you have one, you have `"type": "dotnet"` set for the active configuration. For command-line arguments, "figure out from the project file" means to pull the value from `<Project-Directory>/Propeties/launchSettings.json`. The advantage of `launchSettings.json` is that it allows settings to be shared between Visual Studio Code, full Visual Studio, and `dotnet run`.
```

> For this case, here are the steps to set the command line arguments:

Reviewer: ```suggestion
For this case, here are the steps to set the command-line arguments:
```

> 1. In workspace explorer view, navigate to the directory of the project (.csproj file) you want to launch

Reviewer: ```suggestion
1. In workspace Explorer view, navigate to the directory of the project (.csproj file) you want to launch
```

2. If there isn't a `Properties` directory already, create it
3. If there isn't a `launchSettings.json` file already, create one, you can use the below text as an example

> 4. Change the `commandLineArgs` property to what you would like the command line arguments to be

Reviewer: ```suggestion
4. Change the `commandLineArgs` property to what you would like the command-line arguments to be
```

**Example `launchSettings.json` file**:
```json
{
  "profiles": {
    "MyLaunchProfileName": {
      "commandName": "Project",
      "commandLineArgs": "MyFirstArgument MySecondArgument"
    }
  }
}
```

### Approach 2: `launch.json`

> If you are using the `coreclr` or `clr` debug adapter type in VS Code, command line arguments are stored in your `<workspace_root>/.vscode/launch.json`. To edit them in this case:

Reviewer: ```suggestion
If you are using the `coreclr` or `clr` debug adapter type in VS Code, command-line arguments are stored in your `<workspace_root>/.vscode/launch.json`. To edit them in this case:
```

    }
  },
  "profiles": {
    "EnvironmentsSample": {
      "commandName": "Project",
      "dotnetRunMessages": true,
      "launchBrowser": true,
      "applicationUrl": "https://localhost:7152;http://localhost:5105",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "EnvironmentsSample-Staging": {
      "commandName": "Project",
      "dotnetRunMessages": true,
      "launchBrowser": true,
      "applicationUrl": "https://localhost:7152;http://localhost:5105",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Staging",
        "ASPNETCORE_DETAILEDERRORS": "1",
        "ASPNETCORE_SHUTDOWNTIMEOUTSECONDS": "3"
      }
    },
    "EnvironmentsSample-Production": {
      "commandName": "Project",
      "dotnetRunMessages": true,
      "launchBrowser": true,
      "applicationUrl": "https://localhost:7152;http://localhost:5105",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Production"
      }
    },
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}
```

> ## Profile Properties

Reviewer: ```suggestion
## Profile properties
```

* `commandLineArgs` - The arguments to pass to the target being run.
* `executablePath` - An absolute or relative path to the executable.
* `workingDirectory` - Sets the working directory of the command.

> * `launchBrowser` - Set to true if the browser should be launched.

Reviewer: ```suggestion
* `launchBrowser` - Set to `true` if the browser should be launched.
```
